include/swift/AST/DiagnosticsSema.def

```diff
@@ -1080,6 +1080,8 @@ ERROR(cdecl_throws,none,
 
 ERROR(attr_methods_only,none,
       "only methods can be declared %0", (DeclAttribute))
+NOTE(access_control_in_protocol_more_detailled_info,none,
+	  "protocol requirements implicitly have the same access as the protocol", ())
```

Comment:
Drive-by nits: "detailed" is misspelled here; also, notes in this file are mostly named "something_note" or "note_something" and not "something_more_detailed_info"; finally, this file uses spaces for alignment--please remove the tab and replace with the appropriate number of spaces.